nvm.sh

```diff
@@ -1929,9 +1929,9 @@ nvm_download_artifact() {
   fi
 
   if [ -r "${TARBALL}" ]; then
-    nvm_err "Local cache found: $(nvm_sanitize_path "${TARBALL}")"
+    nvm_echo "Local cache found: $(nvm_sanitize_path "${TARBALL}")"
     if nvm_compare_checksum "${TARBALL}" "${CHECKSUM}" >/dev/null 2>&1; then
-      nvm_err "Checksums match! Using existing downloaded archive $(nvm_sanitize_path "${TARBALL}")"
+      nvm_echo "Checksums match! Using existing downloaded archive $(nvm_sanitize_path "${TARBALL}")"
       nvm_echo "${TARBALL}"
```

Comment:
A separate issue... 

I expect these nvm_echo commands to generate output but they actually do not. When they are nvm_err output is generated however it's generated to stderr even though it is not an error condition. I'd rather have it print nothing than print a non-error to stderr but I'd really like the nvm_echo command to work at this location.

Reply:
stderr is not just for error conditions; stdout, however, is *just* for parseable output.